packages/client/hmi-client/src/views/Home.vue

```diff
@@ -31,7 +54,12 @@ const mockProjects: string[] = ['Mocked Project'];
 						<NewProjectCard />
 					</li>
 					<li v-for="(project, index) in mockProjects" :key="index">
-						<ProjectCard :name="project" />
+						<ProjectCard
+							:name="project.name"
+							@click.exact="openProject(project.id)"
+							@click.meta="openProjectNewTab(project.id)"
+							@click.ctrl="openProjectNewTab(project.id)"
```

Comment:
Why do we need those?
I think those should be links that the router highjack instead.

```vue
<li v-for="project in mockProjects" :key="project.id">
  <a :href="`/projects/${project.id}`">
	  <ProjectCard :name="project.name" />
  </a>
</li>
```

Reply:
Will try it!

Reply:
Oh, I remember why I didn't do this now. Wrapping the project card in A broke the hover effect.